JSQMessagesViewController/Layout/JSQMessagesBubblesSizeCalculator.m

```diff
@@ -138,4 +146,28 @@ - (CGSize)jsq_avatarSizeForMessageData:(id<JSQMessageData>)messageData
     return layout.incomingAvatarViewSize;
 }
 
+- (NSInteger)magixInsetAddition {
```

Comment:
actually, let's make this a property instead. and set it during initialization.

````objective-c
@property (nonatomic, readonly) NSInteger additionalInset;
````